integration-tests/gradle/src/integrationTest/kotlin/org/jetbrains/dokka/it/gradle/AbstractGradleCachingIntegrationTest.kt

```diff
@@ -0,0 +1,135 @@
+package org.jetbrains.dokka.it.gradle
+
+import java.io.File
+import kotlin.test.*
+
+abstract class AbstractGradleCachingIntegrationTest(override val versions: BuildVersions): AbstractGradleIntegrationTest() {
+    fun setupProject(project: File) {
+        val templateProjectDir = File("projects", "it-basic")
+        project.mkdirs()
+        templateProjectDir.listFiles().orEmpty()
+            .filter { it.isFile }
+            .forEach { topLevelFile -> topLevelFile.copyTo(File(project, topLevelFile.name)) }
+
+        File(templateProjectDir, "src").copyRecursively(File(project, "src"))
+        val customResourcesDir = File(templateProjectDir, "customResources")
+        if(customResourcesDir.exists() && customResourcesDir.isDirectory) {
+            val destination = File(project.parentFile, "customResources")
+            destination.mkdirs()
+            destination.deleteRecursively()
+            customResourcesDir.copyRecursively(destination)
+        }
+
+        // clean local cache for each test
+        if (versions.gradleVersion.version == "7.0") {
```

Comment:
If Gradle 7.0 removed the old syntax then this `if` should be for versions `>= 7.0`